grass7/raster/r.fuzzy.system/r.fuzzy.system.html

```diff
@@ -221,102 +225,107 @@ <h2>EXAMPLE</h2>
 </pre></div>
 
-Next, to perform analysis we need two files: one with definition of map used in
-analysis and definition of fuzzy sets for every map, and second with fuzzy
-rules. For this example:
+Next, to perform the analysis we need two files: one with the
+definition of maps used in analysis and the definition of fuzzy sets for
+every map, and a second with fuzzy rules. For this example:
 
-<p>MAPS
+<p>MAPS:</p>
 
 Note: the raster map names are specified with a "%" character (here 
-"%elevation" and "%distance" are the input maps and "%flood" the
-output map:
+"% elevation" and "% distance" are the input maps). The single output map
+must be named "%_OUTPUT_", in the map file, but the actual raster will
+be assigned the name from the "output" option to "r.fuzzy.system"
+(in the example below this is "flood").
 
 <div class="code"><pre>
-#flood.map
-	%elevation
+# flood.map
+% elevation
 $ low {right; 2,6; sshaped; 0; 1}
 $ high {left; 2,6; sshaped; 0; 1}
-	%distance
+% distance
 $ near {right; 40,80; sshaped; 0; 1}
 $ far {left; 40,80; sshaped; 0; 1}
 #output map
-	%flood
+% _OUTPUT_
 $ unprob {both; 0,20,20,40; linear; 0;1}
 $ prob {both; 20,40,40,60; linear; 0;1}
 $ veryprob {both; 40,60,60,80; linear; 0;1}
 </pre></div>
 
 <p>RULES:
 <div class="code"><pre>
-#flood.rul
+# flood.rul
 $ unprob {elevation = high & distance = far}
 $ prob {distance = near | elevation = low}
 $ veryprob {distance = near & elevation = low}
 </pre></div>
 
 finally we need run r.fuzzy.system:
 <div class="code"><pre>
-r.fuzzy.system maps=flood.map rules=flod.rul output=flood 
+r.fuzzy.system maps=flood.map rules=flood.rul output=flood 
 </pre></div>
 
-Resulting map should look like this below. Yellow colour means no risk, red high
+The resulting map should look like the image below. Yellow colour means no risk, red high
 risk, green, blue end so on moderate risk.
 <CENTER><img src="f_result.png" border="1"></CENTER><br>
 
 <h2>SEE ALSO</h2>
 
 <em>
-<a href="r.fuzzy.html">r.fuzzy</a>,
-<a href="r.fuzzy.logic.html">r.fuzzy.logic</a>,
-<a href="r.fuzzy.set.html">r.fuzzy.set</a>,
+<a href="r.fuzzy.logic.html">r.fuzzy.logic</a> addon,
+<a href="r.fuzzy.set.html">r.fuzzy.set</a> addon,
 <a href="r.mapcalc.html">r.mapcalc</a>,
 </em>
 
 <h2>REFERENCES</h2>
 
-Zadeh, L.A. (1965). "Fuzzy sets". Information and Control 8 (3): 338–353.
-doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.<p>
+<ul>
+<li>Jasiewicz, J. (2011). A new GRASS GIS fuzzy inference system for massive data analysis.
+Computers & Geosciences (37) 1525-1531. DOI <a href="http://10.1016/j.cageo.2010.09.008">http://10.1016/j.cageo.2010.09.008</a>
+
+<li>Zadeh, L.A. (1965). "Fuzzy sets". Information and Control 8 (3): 338–353.
+doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.
 
-Nov&aacute;k, Vil&eacute;m (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
-ISBN 0-85274-583-4.<p>
+<li>Novák, Vilém (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
```

Comment:
AFAIK this will fail: the previous HTML codes are needed.

Reply:
Leaving for now. In my experience modern browsers support UTF-8.